client/app/components/project/package-section.js

```diff
@@ -23,8 +23,8 @@ export default class ProjectPackageSectionComponent extends Component {
 
   get invoices() {
     return this.args.packages
-      .mapBy('invoices')
-      .reduce((acc, curr) => acc.concat(curr), []);
+      .map((pkg) => pkg.get('invoices').toArray())
+      .flat();
```

Comment:
Okay so this is alllll invoices across all packages in one array, cool

Reply:
yup! I didn't even know about the `flat` method.

I was running into a weird issue with Ember Data hasMany relationships, though, which is why I had to map and `toArray` stuff. Note to self: Ember Data relationships are NOT normal arrays.